core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -994,6 +1006,139 @@ public void date_format() {
     assertEquals(missingValue(), eval(dsl.date_format(DSL.literal(""), missingRef)));
   }
 
+  private FunctionExpression maketime(Expression hour, Expression minute, Expression second) {
```

Comment:
I realize these tests follow the convention in this file, but the tests would be much easier to understand if each function had it's own test suite, and each test in a suite tested a particular scenario -- all valid arguments, null as an argument, negative values, rounding behaviour, etc.

I'll leave it up to you whether you want to break with the convention and introduce a MakeTimeTests.

Reply:
This is a good suggestion.  I would recommend setting the convention for the test file to include the original name, such as: `DateTimeFunctionMakedateTest` and `DateTimeFunctionMaketimeTest`.  It would be easier to find for future developers.